### ISSUE TITLE
CC a11y audit only - tooltip component

### ISSUE THREAD

**Issue by cory-churches** [uswds/uswds#5452]
Audit the link component for WCAG compliance and critical checklist checks using three manual accessibility testing methods.
[Critical Checklist spreadsheet](https://docs.google.com/spreadsheets/d/1fAJaObQHv7aIaeUpH4fTLmauhmugMtNCrbpZe4BJcPY/edit?usp=sharing)
This ticket will be considered done once the following content has been written and associated WCAG checks are in place for the tooltip component:
WCAG criteria related to [tooltip component](https://designsystem.digital.gov/components/tooltip/) elements
How to perform manual testing (critical checklists) and related user-focused questions for downstream teams to test. a. Zoom mag to 200% b. Keyboard c. JAWS The results will be added into the spreadsheet. You can use this document as a starting reference: https://docs.google.com/document/d/1l8KP9A77rg6IF7YME0CvadMMczZKU5FMHvBqkyWFIfM/edit?usp=sharing
[List of all components is here](https://docs.google.com/spreadsheets/d/1ASzZ80oI6WVWAifqPO6gYAqrsRaiZs_qGTOStuQPyKs/edit?usp=sharing)

**Comment by amycole501** [comment]
[Research findings are here](https://docs.google.com/document/d/1sMQIEpWqnKQOeDlETX2ngFbMMQ4L_tjxTf022oS4mp8/edit?usp=sharing).

**Comment by amycole501** [comment]
There are at least 13 success criteria to review so this ticket will need to move into the next sprint.

**Comment by amycole501** [comment]
12 of 13 success criteria have been checked with one failure (user is unable to use the "escape" key in Chrome to dismiss the tooltip).  One SC was confusing and may not need to be checked/may relate to another already tested criteria. See notes in the document.
